static/versionDetails.json

```diff
@@ -155,7 +155,7 @@
     },
     {
       "flag": "isBlockStorageEnabled",
-      "description": "Use block devices instead of the filesystem for storage in the Ceph cluster.",
+      "description": "Use block devices instead of the filesystem for storage in the Ceph cluster. This flag will be ignored for versions 1.4.3+.",
```

Comment:
"This flag will be automatically set for versions 1.4.3+" is what I would say here

Because we're not ignoring it as much as always taking the behavior it represents